user-stories.md

```diff
@@ -0,0 +1,8 @@
+# Пользовательские истории:
+- Как пользователь, я хочу зарегистрироваться и получить свой аккаунт с уникальным id, чтобы меня можно было отличить от других пользователей.
+- Как пользователь, я хочу получать короткую уникальную ссылку по длинному URL, чтобы было проще её использовать.
+- Как пользователь, я хочу использовать выданную мне короткую ссылку и переходить на соответствующий сайт без задержек.
+- Как зарегистрированный пользователь, я хочу иметь возможность просматривать уже созданные мною ранее сокращения для ссылок, чтобы использовать их повторно.
+- Как зарегистрированный пользователь, я хочу иметь возможность удалять созданные мною ранее сокращения для ссылок.
```

Comment:
You missed motivational part in this story.

Should you specify the behavior of a removed link?